LoopbackGuideApplication/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.10.0'
+        classpath 'com.android.tools.build:gradle:1.5.+'
```

Comment:
I think it would be nice if we could eventually upgrade to gradle 2.x, but let's leave that for another day.